src/lib/core/overlay/overlay.ts

```diff
@@ -34,9 +35,10 @@ export class Overlay {
   constructor(private _overlayContainer: OverlayContainer,
               private _componentFactoryResolver: ComponentFactoryResolver,
               private _positionBuilder: OverlayPositionBuilder,
+              private _scrollStrategyOptions: ScrollStrategyOptions,
```

Comment:
Expose `scrollStrategies` as a public property on `Overlay` so each component won't have to inject it separately?